broker/src/main/java/io/zeebe/broker/transport/backpressure/CommandRateLimiter.java

```diff
@@ -23,8 +25,8 @@ protected CommandRateLimiter(CommandRateLimiterBuilder builder) {
   }
 
   @Override
-  public Optional<Listener> acquire(Void context) {
-    if (getInflight() >= getLimit()) {
+  public Optional<Listener> acquire(Intent intent) {
+    if (getInflight() >= getLimit() && intent == JobIntent.COMPLETE) {
```

Comment:
Are we only rejecting if the limit is in flight is >= limit and the intent is Job COMPLETE? So we never reject anything but job complete commands? Or am I misreading this?

Reply:
Oh my mistake :see_no_evil:  It was `!equals()` before I replaced it with `==`. We should never reject job complete commands even if the limit is reached.